app/app/main.py

```diff
@@ -80,55 +84,127 @@ def validate_extension(filename):
     )
  
 
+def remove_folder(path,tdate):
+
+        if not shutil.rmtree(path):
+                logger.info(f"---{tdate}---{path} folder is removed successfully")
+        else:
+                logger.info(f"---{tdate}---Unable to delete the {path}")
+
+
+def remove_file(path,tdate):
+
+           if not os.remove(path):         
+                logger.info(f"---{tdate}---{path} file is removed successfully")
+           else:           
+                logger.info(f"---{tdate}---Unable to delete the {path}")
+
 
+def get_file_or_folder_age(path):
+
+        # getting modified time of the file/folder
+        # time will be in seconds
+        mtime = os.stat(path).st_mtime
+
+        return mtime
+        
 @app.on_event("startup")
 async def startup():
-    await database.connect()
+        await database.connect()
+
+@app.on_event("startup")
+@repeat_every(seconds= 24 * 60 * 60) 
+async def remove_old_files():
+
+
+    if(env('ENABLE_PRUNER')=='true'):
```

Comment:
We do not need `ENABLE_PRUNER` if the configuration file already details whether or not pruning is enabled for a given directory.

Reply:
nope , i removed the enable option for each directory in the toml. if they want to disable it later they can just delete their entry from the toml. this env var will be set and cannot be changed without rerunning pod-compose.sh

Reply:
to improve readability in the toml file and remove an extra var , if somebody does not want to enable pruner on their directory they can just delete the entry from the toml.

Reply:
it sounds like we don't need an environment variable for anything then

Reply:
even if we decide to keep them  in the toml, we will still need the enable_pruner env var , since not everybody who deploys snappy might want to enable the pruner

Reply:
Vyper-Config assumes there should be a configuration file, but the pruner feature is optional. Using an `ENABLE_PRUNER` environment variable facilitates the current split in configuration between application's shell environment variables and the toml configuration file.